docs/simnet_environment.mediawiki

```diff
@@ -0,0 +1,321 @@
+tickets.
+
+===2.3.3. Rejected ticket purchases with errors similar to <code>rejected transaction X: transaction X expired at height Y</code>===
+
+This is caused by the same things as the previous issue and has the same
+solution.
+
+===2.3.4. Errors similar to <code>list size too small: 0 < 5</code>===
+
+This happens if you stop buying tickets and mine enough blocks to the point the
+chain is unrecoverable due to the ticket pool becoming depleted.  The solution
+is to start over and ensure you are purchasing enough tickets.  A new chain is
+created every time the simnet environment setup script is run.
+
+Note that this possibility is temporary and will be replaced with the next error
+in the future.
+
+===2.3.5. Errors similar to <code>chain continuity not guaranteed after height X</code>===
+
+This happens if you stop buying tickets and mining the next block would cause
+the chain to become unrecoverable if no tickets are included due to depletion of
+the live ticket pool.  The solution is to purchase enough tickets so the block
+can be mined.
+
+==4. Manual Configuration==
+
+'''<p>
+It is _HIGHLY_ recommended to use the previously described preconfigured simnet
+environment instead of manually configuring things.  Not only does it greatly
+simplify things, it also provides a much more sophisticated setup than the
+basic on described here.
+</p>'''
+
+Running a single <code>dcrd</code> node on simnet is simply starting
+<code>dcrd</code> with the <code>--simnet</code> flag.  However, in order to be
+really useful, you'll typically want to be able to send coins amongst addresses
+which implies that blocks will need to be mined and interfacing with a wallet
+will be needed.
+
+In addition, since there are effectively no coins yet on the new private
+network, an initial series of blocks will need to be mined which pay to an
+address you own so there are usable coins to spend.
+
+As previously mentioned, simnet uses unique addresses to prevent confusion with
+the main network.  Thus, it means that a wallet which supports the address
+format must be used.  For this, <code>dcrwallet</code> with the
+<code>--simnet</code> flag can be used.
+
+The following is a command reference to get going:
+
+'''<p>
+NOTE: All of these commands can be simplified by creating config files and
+making use of them, however the commands here use all switches on the
+command line to show exactly what is needed for each.
+</p>'''
+
+* Create a new simnet wallet (this will be called the first window):
+  $ dcrwallet --simnet --create
+  Enter the private passphrase for your new wallet:
+  Confirm passphrase:
+  Do you want to add an additional layer of encryption for public data? (n/no/y/yes) [no]:
+  Do you have an existing wallet seed you want to use? (n/no/y/yes) [no]: y
+  Enter existing wallet seed (followed by a blank line): b280922d2cffda44648346412c5ec97f429938105003730414f10b01e1402eac
```

Comment:
I wonder how long it will take to get the first bug bounty report that we have this secret committed in the repo...

Reply:
I did consider that, but since it's in the script too, I figured it didn't matter.  That said, I probably should change this to generate one and just do a `<seed here>` since this is public facing documentation.

Reply:
Updated to show it being done by generating the seed with placeholders.